SUPPORT.md

```diff
@@ -1,3 +1,5 @@
+# What to do?
```

Comment:
Maybe make the text smaller?

Perhaps replace with something more along the lines of "What should I do?"

Reply:
Eh. Text size is alr. Replaced.